config/initializers/plugin_test.rb

```diff
@@ -0,0 +1,72 @@
+# This initializer allows to run tests from Foreman plugins
+# via rails 5 test runner
+# Usage: bundle exec rails test --engine foreman_bootdisk ../foreman_bootdisk/test/unit/example_test.rb
+require 'rails/test_unit/runner'
```

Comment:
Should we continue with the script just in case it's being run via rails? Just in case there is a chance this would influence production environment?

Reply:
i.e. catch and ignore a load error here?

Reply:
catch an ignore would be fine. Also, as discussed offline, since this actually works only with spring, would be good to cover this here, either with some conditional skip when not running in spring env, or at least in the comment